src/Maths/Silk.NET.Maths/Scalar.Constants.cs

```diff
@@ -87,6 +87,22 @@ public static partial class Scalar<T> where T : unmanaged
         /// </summary>
         public static readonly T Tau;
 
+        /// <summary>
+        /// Represents the number of degrees in a single radian.
+        /// </summary>
+        /// <remarks>
+        /// This is equivalent to <c>180 / MathF.Pi</c>
+        /// </remarks>
+        public static readonly T DegreesPerRadian;
+
+        /// <summary>
+        /// Represents the number of radians in a single degree.
+        /// </summary>
+        /// <remarks>
+        /// This is equivalent to <c>MathF.Pi / 180</c>.
+        /// </remarks>
+        public static readonly T RadiansPerDegree;
```

Comment:
These seem to be missing from the PublicAPI.Shipped.txt

Reply:
They are not.